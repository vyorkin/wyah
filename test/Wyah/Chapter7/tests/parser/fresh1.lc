-- Fresh variables

let wtf = \a b c d e e' f g h i j k l m n o o' o'' o''' p q r r' s t u u' v w x y z ->
    q u i c k b r o w n f o' x j u' m p s o'' v e r' t h e' l a z y d o''' g;
