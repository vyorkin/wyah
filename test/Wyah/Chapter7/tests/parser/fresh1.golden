Illegal character at 3:24 on char '\'' before : 'f g h i j k l m n o o' o'' o''